desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -177,12 +186,31 @@ def _make_adls_request(self, request_data, path, resource_path):
           "container": container,
           "relativepath": relative_path,
         },
-        "resourceOwner": storage_account,
-        "action": "read",
-        "accessTypes":["read"]
+        "action": access_type,
+        "accessTypes":[access_type]
       }
     })
 
+
+  def handle_adls_req_mapping(self, method, params, relative_path):
+    access_type = ''
+
+    if method == 'HEAD':
```

Comment:
FYI: a good way going forward to handle the numerous types would be to build a dict mapping of the action --> access_type